lib/audited/auditor.rb

```diff
@@ -63,7 +63,10 @@ def audited(options = {})
         has_many :audits, ->(audited_record) do
           where(
             service_name: Rails.application.class.parent_name,
-            created_at: (audited_record.created_at - 1.minute)..(Time.now)
+            created_at: Range.new(
+              ((audited_record.created_at || Time.now)  - 1.day),
```

Comment:
LGTM. 
Minor comment: did you run rubocop against your changes? Its just doubled space here

Reply:
Good eye. Fixing it.

Reply:
There is no rubocop config specified and not present ad development dependency, I don't want to autocorrect whole file as we are updating with official repository and that would generate a lot of unnecessary conflicts